.github/workflows/jira-lint-and-link.yml

```diff
@@ -7,13 +7,17 @@ jobs:
   add-jira-description:
     runs-on: ubuntu-latest
     # Run only if the PR is not from a Fork / external contributor
-    if: (!startsWith(github.ref, 'refs/heads/dependabot/') && github.repository_owner == 'wireapp')
+    if: ${{ github.repository_owner == 'wireapp' }}
```

Comment:
@emil-wire this looks fine to me. I don't thing this can be injected from the outside

Reply:
lgtm